quill-cassandra/src/main/scala/io/getquill/CassandraAsyncContext.scala

```diff
@@ -48,7 +48,7 @@ class CassandraAsyncContext[N <: NamingStrategy](
     executeQuery(cql, prepare, extractor).map(handleSingleResult)
 
   def executeAction[T](cql: String, prepare: Prepare = identityPrepare)(implicit ec: ExecutionContext): Future[Unit] = {
-    Future(prepare(this.prepare(cql))).flatMap {
+    this.prepareAsync(cql).map(prepare).flatMap {
```

Comment:
I'm thinking of moving `prepare` into `prepareAsync` call or in `flatMap`, which can reduce additional call to `ec`